examples/Dockerfile-envoy-image

```diff
@@ -4,3 +4,6 @@ ADD build/source/exe/envoy /usr/local/bin/envoy
 RUN apt-get update && apt-get install -y \
     curl \
     python-pip
+RUN objcopy --only-keep-debug /usr/local/bin/envoy /usr/local/bin/envoy.dbg
```

Comment:
For now I would just strip and drop the debug symbols to make the image smaller. If people want the debug symbols we can deal with that later. Also, do we need curl and python above? Can that be deleted?